packages/mesos-modules/buildinfo.json

```diff
@@ -3,7 +3,7 @@
     "single_source" : {
       "kind": "git",
       "git": "https://github.com/dcos/dcos-mesos-modules.git",
-      "ref": "8d09c2d44f606df23a57254ebcbc5f2722a3b677",
-      "ref_origin": "master"
+      "ref": "86f14f9402e324f78385a0b7b9509079dda7c0ee",
+      "ref_origin": "1.8.x"
```

Comment:
Why does the new dcos-mesos-modules self-identify as `AC_INIT([dcos-mesos-modules],[0.2])` (same as what's in master/1.9)? Should this be 0.1 instead, or 1.8.8? How do we want to version these modules? I'd suggest we follow along with the DC/OS version number they're targeted for (1.8.8 vs. 1.9.0)